wd_core.au3

```diff
@@ -1285,13 +1285,12 @@ Func _WD_Startup()
 
 		__WD_ConsoleWrite($sFuncName & ": OS:" & @TAB & @OSVersion & " " & @OSType & " " & @OSBuild & " " & @OSServicePack)
 		__WD_ConsoleWrite($sFuncName & ": AutoIt:" & @TAB & @AutoItVersion)
-		__WD_ConsoleWrite($sFuncName & ": au3WD UDF:" & @TAB & $__WDVERSION & $sUpdate)
+		__WD_ConsoleWrite($sFuncName & ": WebDriver UDF:" & @TAB & $__WDVERSION & $sUpdate)

```

Comment:
This is already changed in my other PR

Reply:
Yes. I know. I just want to keep it the same.

Reply:
I revert it back and make speparate bigger PR.

Reply:
As you can see wd_core.au3 is not changed in this PR anymore.